addons/website/static/src/js/website.snippets.editor.js

```diff
@@ -1308,6 +1308,107 @@
             this.$target.find('.item.text_image, .item.image_text, .item.text_only').find('.container > .carousel-caption > div, .container > img.carousel-image').attr('contentEditable', 'true');
         },
     });
+
+    website.editor.countdown = website.editor.Dialog.extend({
+        template: 'website.countdown_configuration',
+        events : _.extend({}, website.editor.Dialog.prototype.events, {
+            'click button[data-action=set_countdown]': 'set_countdown',
+        }),
+        init: function (parent) {
+            this.$target = parent.$target;
+            return this._super();
+        },
+        start: function () {
+            var self = this;
+            this.date_input = self.$el.find("#web_countdown_date").datepicker({
+                minDate: 0,
+            });
+            return this._super();
+        },
+        set_countdown: function () {
+            var self = this;
+            var release_date = this.date_input.datepicker( 'getDate' );
+            var hour = document.getElementById("web_countdown_hour").value;
+            var minute = document.getElementById("web_countdown_min").value;
+            var second = document.getElementById("web_countdown_sec").value;
```

Comment:
you can use jQuery("#ID").val() instead of getElementById, for the sake of readability